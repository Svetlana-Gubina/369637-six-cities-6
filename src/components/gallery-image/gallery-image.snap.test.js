import React from 'react';
import {render} from '@testing-library/react';
import GalleryImage from './gallery-image';

test(`Should GalleryImage render correctly`, () => {
  const url = `img/1.png`;

  const {container} = render(<GalleryImage imageUrl={url} />);
  expect(container).toMatchSnapshot();
});
