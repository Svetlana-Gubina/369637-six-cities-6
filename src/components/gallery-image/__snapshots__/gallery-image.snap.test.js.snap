// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should GalleryImage render correctly 1`] = `
<div>
  <div
    class="property__image-wrapper"
  >
    <img
      alt="Photo studio"
      class="property__image"
      src="img/1.png"
    />
  </div>
</div>
`;
